build.xml

```diff
@@ -14,7 +14,7 @@
   <import file="build-res/subfloor-pkg.xml"/>
 
   <!-- Every non-api shim module. These will be compiled/resolved after api. -->
-  <property name="shim-non-api-modules" value="hive-jdbc,hadoop-20,cdh42,cdh50,hdp13,hdp20,hdp21,mapr21,mapr30,mapr31" />
+  <property name="shim-non-api-modules" value="cdh42" />

```

Comment:
This change should be reverted, you can override this property from the command-line if you wish